
since it is included from clutter/cluttermm/Makefile* and also contains wrap_init.h already

diff -Nur cluttermm-0.9.4-old/build_shared/Makefile_build_gensrc.am_fragment cluttermm-0.9.4/build_shared/Makefile_build_gensrc.am_fragment
--- cluttermm-0.9.4-old/build_shared/Makefile_build_gensrc.am_fragment	2009-10-04 02:44:05.094884570 +0300
+++ cluttermm-0.9.4/build_shared/Makefile_build_gensrc.am_fragment	2009-10-04 02:44:24.914886641 +0300
@@ -21,6 +21,6 @@
 
 files_built_cc		= $(files_hg:.hg=.cc) wrap_init.cc
 files_built_h		= $(files_hg:.hg=.h)
-files_extra_h		+= wrap_init.h
+#files_extra_h		+= wrap_init.h
 
 include $(top_srcdir)/build_shared/Makefile_build.am_fragment
